devel/d3dmetal/Portfile

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                d3dmetal
+version             1.1
+epoch               0
+categories          devel
+maintainers         {@Gcenx gmail.com:gcenx83}
+homepage            https://developer.apple.com/games
+platforms           {darwin any >= 23}
+license             Restrictive
+supported_archs     x86_64
+description         Direct3D to Metal translation layer
+long_description    {*}${description}
+master_sites        https://download.developer.apple.com/Developer_Tools/Game_Porting_Toolkit_${version}/Game_Porting_Toolkit_${version}.dmg
```

Comment:
`master_sites` is for the URL of the directory that the distfile is in. It does not include the distfile name.

Reply:
What I said is still accurate.